frontend/src/clients/board/BoardClient.ts

```diff
@@ -8,6 +8,9 @@ export class BoardClient extends HttpClient {
   response = {}
 
   getVerifyBoard = async (params: BoardRequestDTO) => {
+    this.isBoardVerify = false
+    this.isNoDoneCard = false
```

Comment:
`isNoDoneCard`, maybe using `haveDoneCard` is good than yours.

Reply:
Done